editor/src/editor/layout/inspector/material/pbr.tsx

```diff
@@ -225,18 +228,37 @@ export class EditorPBRMaterialInspector extends Component<IEditorPBRMaterialInsp
+
+				<EditorInspectorSectionField title="Misc">
 					<EditorInspectorSwitchField label="Unlit" object={this.props.material} property="unlit" />
 					<EditorInspectorSwitchField label="Disable Lighting" object={this.props.material} property="disableLighting" />
 					<EditorInspectorSwitchField label="Use Specular Over Alpha" object={this.props.material} property="useSpecularOverAlpha" />
 					<EditorInspectorSwitchField label="Enable Specular Anti Aliasing" object={this.props.material} property="enableSpecularAntiAliasing" />
 					<EditorInspectorSwitchField label="Force Irradiance In Fragment" object={this.props.material} property="forceIrradianceInFragment" />
 					<EditorInspectorSwitchField label="Use Radiance Occlusion" object={this.props.material} property="useRadianceOcclusion" />
 					<EditorInspectorSwitchField label="Use Physical Light Falloff" object={this.props.material} property="usePhysicalLightFalloff" />
+					<EditorInspectorSwitchField label="Use GLTF Light Falloff" object={this.props.material} property="useGLTFLightFalloff" />
+					<EditorInspectorSwitchField label="Use Radiance Over Alpha" object={this.props.material} property="useRadianceOverAlpha" />
+					<EditorInspectorSwitchField label="Use Object Space Normal Map" object={this.props.material} property="useObjectSpaceNormalMap" />
+					<EditorInspectorSwitchField label="Use Horizon Occlusion" object={this.props.material} property="useHorizonOcclusion" />
 					<EditorInspectorSwitchField label="Separate Culling Pass" object={this.props.material} property="separateCullingPass" />
+					<EditorInspectorSwitchField label="Use Lightmap As Shadowmap" object={this.props.material} property="useLightmapAsShadowmap" />
+					<EditorInspectorSwitchField label="Disable Bump Map" object={this.props.material} property="disableBumpMap" />
+					<EditorInspectorSwitchField label="Use Alpha From Albedo Texture" object={this.props.material} property="useAlphaFromAlbedoTexture" />
+					<EditorInspectorSwitchField label="Force Alpha Test" object={this.props.material} property="forceAlphaTest" />
+					<EditorInspectorSwitchField label="Use Only Metallic From Metallic Reflectance Texture" object={this.props.material} property="useOnlyMetallicFromMetallicReflectanceTexture" />
 					<EditorInspectorNumberField label="Z Offset" object={this.props.material} property="zOffset" />
+					<EditorInspectorNumberField label="Z Offset Units" object={this.props.material} property="zOffsetUnits" />
+					<EditorInspectorNumberField label="Point Size" object={this.props.material} property="pointSize" min={0} />
+					<EditorInspectorSwitchField label="Points Cloud" object={this.props.material} property="pointsCloud" />
+					<EditorInspectorSwitchField label="Fog Enabled" object={this.props.material} property="fogEnabled" />
+					<EditorInspectorNumberField label="Max Simultaneous Lights" object={this.props.material} property="maxSimultaneousLights" min={0} />
```

Comment:
This is managed already by the editor.
Point lights don't include lights that are not in the "range" value
Others will benefit later to "excluded" or "included" meshes list

Reply:
Removed!